### PR TITLE
Print styling

### PR DESCRIPTION
**Pull request by KyraWayne**
will style printing for readability

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -76,3 +80,11 @@ th {
     width: 100%;
     margin-bottom: 10px;
 }
+
+h5{
+    display:none;
+    visibility:hidden;
```
Comment (jkingsman) [inline css/style.css:86]: Could we add a custom class or something to this? Making an unguarded `display: none` on a standard style seems dangerous to me.

File: js/print.js [hunk 2]
```diff
@@ -1,3 +1 @@
-/*window.onafterprint = function(){
-    document.getElementsByClass
-}*/
\ No newline at end of file
+$('.print-btn').on('click',$('ul.unscheduledList li').append(', '));
```
Comment (jkingsman) [inline js/print.js]: Is this still needed with the CSS work you've done?

**Review by jkingsman — CHANGES_REQUESTED** [review]
Looks good; a couple questions and 1 or 2 changes requested but all minor.

File: js/print.js (hunk 2 above)
Comment (KyraWayne) [reply js/print.js]: deleted the file

File: css/style.css (hunk 1 above)
Comment (KyraWayne) [reply css/style.css:86]: added .print-titles class

**Pull request merged**